views/pages/index.ejs

```diff
@@ -71,9 +71,9 @@
     <footer>
       <nav>
         <ul class="flex wrap small">
-          <li class="p-a-1"><a href="https://opencollective.com/weather-10kb?wxkb.io" rel="noopener nofollow" target="_blank">wxkb.io finances</a></li>
-          <li class="p-a-1"><a href="https://github.com/JulianNorton/weather-10kb?wxkb.io" rel="noopener" target="_blank">wxkb.io Github</a></li>
-          <li class="p-a-1"><a href="https://darksky.net/dev/?wxkb.io" rel="noopener nofollow" target="_blank">Powered by Dark Sky</a></li>  
+          <li class="p-a-1"><a href="https://opencollective.com/weather-10kb?wxkb.io" rel="noopener nofollow" target="_blank" title="Link opens in a new window" class="blank">wxkb.io finances</a></li>
```

Comment:
TBH I was following convention as the utility classes aren't particularly semantic, but I'll change the class name to `target-blank`.